query/subscribeEthGovernanceVotesByProposalId.graphql

```diff
@@ -0,0 +1,10 @@
+subscription SubscribeEthGovernanceVotesByProposalId($proposalId: String!) {
+  councilVoteHistories(where: { proposal: { id_eq: $proposalId } }, limit: 50) {
```

Comment:
add orderBy: blockNum_DESC. see other query for a correct way

Reply:
done.